src/style.css

```diff
@@ -109,3 +109,63 @@ footer {
   font-family: 'CreativeThoughts', sans-serif;
   font-size: 20px;
 }
+
+.comments-popup {
+  display: flex;
+  position: fixed;
+  flex-direction: column;
+  gap: 20px;
+  align-items: center;
+  border: 1px solid;
+  z-index: 11;
+  box-sizing: border-box;
+  width: 90%;
+  margin: auto;
+  height: 90vh;
```

Comment:
- [ ] Optional  suggestions

Kindly recommend you increase the height of your popup window